packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
On Wed, 12 Aug 2015, Christophe Troestler wrote:

> oasis2opam generates this complex dependency because it cannot figure out which
> versions of the OPAM package camlp4 actually provide the camlp4 library. If there were
> "findlib" files in all camlp4 versions in opam-repository, there would not be such long
> dependency.

Ok, thanks. So no need to report it to oasis2opam I guess.

I've fixed the opam file as suggested by simplifying the camlp4 deps, 
and now have completely different errors in the build tests :

https://travis-ci.org/ocaml/opam-repository/builds/75397049
https://travis-ci.org/ocaml/opam-repository/builds/75056483

Since it is the same tgz, I'm a bit confused.

Actually I get various kind of errors on the different builds. I'm 
completely confused.
-- 
R1.